CHANGELOG.md

```diff
@@ -144,19 +37,24 @@ testing and greater compatibility with dependent projects.
   WebSocketClient for more robust event subscriptions. It can be enabled with the
   `websocket-client` feature. Subscriptions are exposed using unbounded
   channels. ([#516])
-  
-- `[tendermint]` Removed all trace of Amino, including `amino_types` modules. 
-   All types are now `DomainType`s implementing Protobuf-encoding using Prost. 
-   ([#504], [#535], [#536], [#585])
-
-- `[tendermint]` Protocol breaking changes for compatibility with Tendermint Core v0.34 (and the Cosmos Stargate release) ([#305]):
-    - Validators are now sorted by voting power (descending) 
+- `[tendermint]` Removed all traces of Amino, including `amino_types` modules.
+  All types are now `DomainType`s implementing Protobuf-encoding using Prost.
+  ([#504], [#535], [#536], [#585])
+- `[tendermint]` Protocol breaking changes for compatibility with Tendermint
+  Core v0.34 (and the Cosmos Stargate release) ([#305]):
+  - Validators are now sorted by voting power (descending)
     and address (ascending). ([#506])
-    - Remove PubKey field from DuplicateVoteEvidence ([#502])
-    - Fix hash of empty Merkle tree to comply with RFC6962 ([#498])
-    - All binary encoding is done via protobuf3 instead of amino 
-   ([#504], [#535], [#536], [#585])
-    - Various updates to JSON encoding ([#505])
+  - Remove PubKey field from DuplicateVoteEvidence ([#502])
+  - Fix hash of empty Merkle tree to comply with RFC6962 ([#498])
+  - All binary encoding is done via protobuf3 instead of amino
+    ([#504], [#535], [#536], [#585])
+  - Various updates to JSON encoding ([#505])
+- `[tendermint]` Direct serialization capabilities have been removed from the
+  domain types. ([#639])
+- `[tendermint]` Work has started on making it compulsory to construct domain
+  types by way of their constructors to ensure validity. ([#639])
+- `[tendermint-proto]` The `DomainType` trait has now been renamed to
+  `Protobuf` to clarify its purpose throughout the codebase. ([#672])
```

Comment:
```suggestion
```

Reply:
Good catch! I wonder how this happened...